js/src/modals/WalletSettings/walletSettings.js

```diff
@@ -51,12 +52,27 @@ class WalletSettings extends Component {
       return (
         <Modal
           visible
-          title='rejected'
+          title={
+            <FormattedMessage
+              id='ui.walletSettings.rejected.title'
```

Comment:
Can we just make it `walletSettings.*`? `ui.*` is for components under `~/ui` - will just flatten out the produced files as well.